blockchain/v2/processor.go

```diff
@@ -53,7 +45,8 @@ type pcStop struct {
 type pcFinished struct {
 	priorityNormal
 	height       int64
-	blocksSynced int64
+	blocksSynced int
```

Comment:
It might be useful writing short comments on what these type represent.

Reply:
Agreed, created a follow up issue: https://github.com/tendermint/tendermint/issues/4201